README.md

```diff
@@ -14,33 +14,22 @@
 
 ## <a name="overview"></a>Overview
 
-### Prime Directive
+The Nerdery’s perspective is that code exists to solve a problem. We favor solutions which are pragmatic and innovative over adherence to any one approach, abstraction, or toolset. Because the front-end engineering ecosystem is constantly evolving, a set of guidelines for working within it must continue to be relevant even as specific implementations change. We prioritize interoperability when choosing solutions, and use our guiding principles to write code that is resilient and easy to maintain. We also look to the practices of the front-end community as a whole to create a system of universal consistency.
 
-The standards herein are a guide to keep developers at the Nerdery interoperable amongst each other and able to produce the highest quality code given the timeline and budget constraints of a project.
+This is a living document and is reflective of the opinions and expertise of The Nerdery’s front-end engineering group. However, the discussion of, contribution to, and evolution of this document is reliant on the front-end community both inside and outside of The Nerdery.
 
-For a successful project, always favor solutions that benefit the project and end users over producing "perfect code".
+### Guiding Principles
 
-Always aim to meet the standards knowing you'll have to break the rules at some point. Be thoughtful about when you deviate and check with a senior software engineer or a principal software engineer for guidance.
+* Code should first and foremost be resilient.
+* Solutions should focus on the best approach for a given problem space, and should not be limited by adherence to implementation details.
+* Discretion should be exercised when selecting tools to solve a problem, preferring those that inherently resonate with the methodology over those that do not.
+* The methodology should be agnostic to the development context in which it is being applied.
+* Consider best practices as thematic and conceptual guidelines, not literal dictates.
+* Consider code to be both an artifact that promotes adoption and suggests best practices in its composition.
 
-### Expectations
 
-* Understand and follow the standards outlined in this document.
-* Understand and follow the [methodology](/standards/methodology.md/).
-* Write code knowing that someone else might be maintaining it.
-* Select tools that benefit the project.
+### Standards
+The standards are hard rules which should be enforceable via automated tools like linters; they are the base upon which consistent and interchangeable code is written.
```

Comment:
It does make sense to me, but do you think a form like this would be more clear @nategeslin? 

`The standards are hard rules: they are the base upon which consistent and interchangeable code is written, and should be enforceable via automated tools (linters).`